### PR TITLE
DB-9213 Log UUID in EndFetching event

### PR DESCRIPTION
**Pull request by ascend1**

### CODE REVIEW

File: db-client/src/main/java/com/splicemachine/db/client/am/ResultSet.java [hunk 1]
```diff
@@ -153,7 +156,7 @@
     public boolean isAfterLast_ = false;
     public boolean isFirst_ = false;
     public boolean isLast_ = false;
-    public boolean rowsetContainsLastRow_ = false;
+    //public boolean rowsetContainsLastRow_ = false;
```
Comment (dgomezferro) [inline db-client/src/main/java/com/splicemachine/db/client/am/ResultSet.java:159]: Can you remove these lines rather than commenting them out?

Comment (ascend1) [reply db-client/src/main/java/com/splicemachine/db/client/am/ResultSet.java:159]: OK, for the lines in `getRef()` and `getArray()`, should I remove them, too? Not sure if we would need them later.

Comment (ascend1) [reply db-client/src/main/java/com/splicemachine/db/client/am/ResultSet.java]: Done.

**Comment by ascend1** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**